.gitignore

```diff
@@ -53,4 +53,4 @@ admiral*.tgz
 .DS_Store
 ._.DS_Store
 
-.Rprofile
\ No newline at end of file
+.Rprofile
```

Comment:
I don't think we need to re-include the Rprofile again? unless I missed something